core/src/main/java/org/jdbi/v3/core/internal/Enums.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core.internal;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.util.Set;
+import org.jdbi.v3.core.EnumByName;
+import org.jdbi.v3.core.EnumByOrdinal;
+import org.jdbi.v3.core.EnumConfig;
+import org.jdbi.v3.core.config.ConfigRegistry;
+import org.jdbi.v3.core.generic.GenericTypes;
+import org.jdbi.v3.core.qualifier.QualifiedType;
+import org.jdbi.v3.core.qualifier.Qualifiers;
+
+public class Enums {
```

Comment:
@TheRealMarnes Just a note on design philosophy of config classes that I'm not sure we've communicated to you:

We've been naming most config classes with pluralized names so that inline usage with the `Configurable.configure()` method reads as "configure row mappers", "configure arguments", etc. So I think I'd like to rename `EnumConfig` to `Enums`. Thus:

```
handle.createUpdate(sql)
    .configure(Enums.class, config -> config.setDefaultEnumStrategy(EnumStrategy.BY_ORDINAL))
    .execute();
```